lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -92,6 +92,93 @@ html.video-fullscreen {
       display: inline;
     }
   }
+
```

Comment:
OK, remove the that big block as it's defined elsewhere. I do need to keep the scope of the .btn-pl-primary wide as it needs to be applied to the timer bar which appears outside of the courseware wrapper. Maybe you can show me a tip?